generate_openj9_scc.sh

```diff
@@ -70,9 +44,21 @@ function check_to_download_tomcat() {
 
         # Downloading tomcat
         if curl --fail -o "${DOWNLOAD_PATH_TOMCAT}"/tomcat.tar.gz "${TOMCAT_DWNLD_URL}"; then
-            # Extracting tomcat
-            tar -xvzf "${DOWNLOAD_PATH_TOMCAT}"/tomcat.tar.gz -C "${INSTALL_PATH_TOMCAT}" --strip-components=1
+            # Verifying checksum
+            echo "${TOMCAT_CHECKSUM} *${DOWNLOAD_PATH_TOMCAT}/tomcat.tar.gz" | sha512sum -c
+            if [ $? != 0 ]; then
+                echo "WARNING: Checksum Mismatch. SCC not generated"
+                # Remove the tar file and installation folder
```

Comment:
The comment here should indicate that we do not fail the docker build if the SCC generation fails for whatever reason. Same for the comment below.

Reply:
Added comments at both the places where we proceed with build even if SCC fails